core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java

```diff
@@ -161,18 +161,16 @@ public void testComplete(TestResult result) {
             @Override
             public void runComplete(TestRunResults results) {
                 firstRun = false;
-                if (results.getCurrentFailing().isEmpty()) {
+                if (results.getCurrentTotalCount() == 0) {
+                    lastStatus = "\u001B[33mNo tests found\u001b[0m";
+                } else if (results.getFailedCount() == 0 && results.getPassedCount() == 0) {
+                    lastStatus = String.format("\u001B[33mAll %d tests were skipped\u001b[0m", results.getSkippedCount());
+                } else if (results.getCurrentFailing().isEmpty()) {
                     lastStatus = String.format(
                             "\u001B[32mAll %d tests are passing (%d skipped), %d tests were run in %dms.\u001b[0m",
                             results.getPassedCount(),
                             results.getSkippedCount(),
                             results.getCurrentTotalCount(), results.getTotalTime());
-                    log.info(
```

Comment:
@stuartwdouglas do you really think it pollutes? Without it, we have no trace of the successful run, maybe we could make it one lined?

Reply:
I don't think we need any trace of a successful run. We have really tried to minimize logging in Quarkus to only display things that are useful, and I don't think this is useful. While you are coding this has the potential to really fill up the logs with stuff that does not really matter.

Reply:
We settled in Zulip with a solution in between, to have the log displayed only when switching from failure to success.